tutorials/tutorial00_flow.ipynb

```diff
@@ -0,0 +1,115 @@
+    "\n",
+    "_Flow already contains a dozen of pre-defined scenarios and their corresponding environments that you can re-use or use as models to build your own scenarios and environments._\n",
+    "\n",
+    "Once you have defined these two classes, the last step is to set up the parameters of the simulation, for instance:\n",
+    "\n",
+    "- number of iterations (for the RL algorithm), number of rollouts (simulations) in each iteration, horizon (number of steps in each rollout)\n",
+    "\n",
+    "- vehicles to add in the network (human vehicles, RL vehicles, vehicle inflows...), color sequences for traffic lights (if there are any)\n",
+    "\n",
+    "- Flow parameters: name of the simulation, scenario and environment classes to be used, simulator to use (Sumo, Aimsun...)\n",
+    "\n",
+    "- RL algorithm to use (PPO, TRPO...), parameters of the algorithm (discount rate, neural network parameters...), number of CPUs/GPUs to use, checkpoint frequency, whether or not to restart from an old checkpoint (to resume training or do transfer learning)\n",
+    "\n",
+    "- whether or not to render the simulation (if you don't see anything happening in the simulator when you run the experiment, it is because rendering has been disabled, which makes training significantly faster)\n",
+    "\n",
+    "It is also possible to run a simulation without doing any training, in this case you won't need any environment or RL algorithm.\n",
+    "\n",
+    "Flow has a lot of examples setting up all these simulation parameters, so you can simply take one of them and modify it according to your needs. Once this is done, you can execute the Python file where the experiment is set up and the training shall begin. \n",
+    "\n",
+    "During the training or after it has ended, you can use Flow's visualization tools in order to visualize the data saved in the checkpoint files generated during the training. You can see how well your agent is doing by running a new simulation in the simulator, that will used the trained policy (this time, the simulation will be rendered). You can also plot the reward or return functions, time-space diagrams, capacity diagrams etc.\n",
+    "\n",
+    "In the next section, we will give an overview of how Flow's codebase is organized, so that you can have some reference marks when you go through the following tutorials."
```

Comment:
```suggestion
    "In the next section, we will give an overview of how Flow's codebase is organized, so that you can have some reference points when you go through the following tutorials."
```